### PR TITLE
Bxmsdoc 2103 master - Adding final detail to the Technical Preview Chapter of the RNs

### PR DESCRIPTION
**Pull request by michelehaglund**
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-2103/#ba-dm-rn-tech-preview-con
Original Jira: https://issues.jboss.org/browse/BXMSDOC-2103

### CODE REVIEW

File: docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc [hunk 1]
```diff
@@ -18,32 +15,48 @@ For more information on Red Hat Technology Preview features support scope,
...
-.Enabling the Deployments perspective
-. Go to *Settings* -> *Roles*.
-. Select the role that you want to enable. Note that the Deployments perspective is available only for the admin, analyst, and developer roles.
-. In the Permissions section, click the arrow next to Perspectives.
-. Do one of the following:
-+
-* If the Deployments perspective is listed, click *Read* next to Deployments.
-* If the Deployments perspective is not listed, select it from the *Select Perspective* menu, then click *Read* next to Deployments.
-* If the *Select Perspective* menu is not visible, click *Add Exception* to make is visible, select *Deployments* from the *Select Perspective* menu,  then click *Read* next to Deployments.
+* Aliases - You can use aliases instead of container IDs in the remote REST APIs of the Planner Server.
```
Comment (etirelli) [inline docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc]: "Planner Server and Decision Server"

File: docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc [hunk 2]
```diff
@@ -18,32 +15,48 @@ For more information on Red Hat Technology Preview features support scope,
...
+* Decision Model and Notation (DMN) Profiles
+//DMN Authoring - The person you are looking for is Michael Anstis, but also to my knowledge this is indeed not included in the Decision Central/Workbench for v7.0
+//DMN Profiles - This is quite an internal-mechanism/advanced thing for now, in my view.
+//One reason a user might want to be aware of DMNProfile, is in relation with Signavio as we implemented a DMNProfile to cover partially of their custom/proprietary DMN constructs. But as we don't have anyone yet in Signavio to confirm and test our implementation, I would prefer not to mention this Signavio DMNProfile in our documentation until the work is complete. Another reason a user might need this, is to disable our helper functions and have the engine 100% adherent to the DMN specification. This feature can be disabled using the system/kmodule property "org.kie.dmn.strictConformance" set to true. A final reason a developer might need this, is to plug-in their own extension, but this is advanced and documented on the directly API.
```
Comment (etirelli) [inline docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc]: please remove lines 24-26

File: docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc [hunk 3]
```diff
@@ -18,32 +15,48 @@ For more information on Red Hat Technology Preview features support scope,
...
+
+//Mario Fusco - Hi Michele, I agree with Lukas' comment: Rule units, Multithreaded engine and Executable model are all experimental features even if with different levels of maturity. In particular while the first 2 are ready to be used (even if we don't provide any support for them yet) the Executable model is still a work in progress. It is not completed and not really usable at the moment. I don't know if it makes sense to underline this difference (or mentioning the Executable model at all at this stage). I'm also CC'ing Mark Proctor in case he has something to add to this. Regards, Mario
+//Mark Proctor-to me, Mario, Lukas
+//Yes leave them in them in the experimental list, but do emphasise that while these have a degree of stability in them (shouldn't crash), we need more time to have potential API changes.
+//Mark Proctor - Yes, they are experimental and subject to api changes. The are maturing beta quality features.
```
Comment (etirelli) [inline docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc]: Can you please edit lines 43-48 to remove people's names?

Comment (michelehaglund) [reply docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc]: Done. Thank you Edson :)

Comment (michelehaglund) [reply docs/product-release-notes/src/main/asciidoc/ba-dm-rn-tech-preview-con.adoc]: Done.

**Pull request merged**